### PR TITLE
Feature/import assets

### PR DESCRIPTION
**Pull request by Manuel-Nunes**

### CODE REVIEW

**Review by Palesa-Mpe — APPROVED** [review]

File: Assets/Scripts/PlayerTPLocationDefinition.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerTPLocationDefinition : MonoBehaviour
```
Comment (raagakrishna) [inline Assets/Scripts/PlayerTPLocationDefinition.cs]: Can you use the full word Teleport here? 
So `PlayerTeleportLocationDefinition`?

**Review by raagakrishna — CHANGES_REQUESTED** [review]

File: Assets/Scripts/PlayerMovement.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        
+        if (Physics2D.IsTouchingLayers(PlayerTP.Collider,LayerMask.GetMask("GameWalls"))) {
+            return;
+        }
+            
+        this.transform.position = getTPLocation( this.gameObject, isInPast);
+        isInPast = !isInPast;
+    }
+
+    private static Vector3 getTPLocation( GameObject player, bool isInPast){
```
Comment (raagakrishna) [inline Assets/Scripts/PlayerMovement.cs]: Can we rename the function to `GetTeleportLocation`?

File: Assets/Scripts/PlayerMovement.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.PlayerLoop;
+
+public class PlayerMovement : MonoBehaviour
+{
+    public PlayerTPLocationDefinition PlayerTP;
```
Comment (raagakrishna) [inline Assets/Scripts/PlayerMovement.cs]: Can we make the variable name here to `PlayerTeleport`?

File: Assets/Scripts/PlayerMovement.cs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.PlayerLoop;
+
+public class PlayerMovement : MonoBehaviour
+{
+    public PlayerTPLocationDefinition PlayerTP;
+    public static float SpeedMultiplier = 10;
+    
+    public bool isInPast = true;
```
Comment (raagakrishna) [inline Assets/Scripts/PlayerMovement.cs]: It looks like you are using CamelCase for public variables, then can we make this `IsInPast`?

**Review by raagakrishna — CHANGES_REQUESTED** [review]

File: Assets/Scripts/PlayerMovement.cs (hunk 2 above)
Comment (raagakrishna) [reply Assets/Scripts/PlayerMovement.cs]: Any feedback on this comment?

File: Assets/Scripts/PlayerMovement.cs (hunk 4 above)
Comment (raagakrishna) [reply Assets/Scripts/PlayerMovement.cs]: Any feedback on this comment?

**Review by raagakrishna — CHANGES_REQUESTED** [review]

**Review by raagakrishna — APPROVED** [review]

**Pull request merged**